crates/cxx-qt/src/lib.rs

```diff
@@ -12,23 +17,23 @@ use cxx_qt_gen::{write_rust, GeneratedRustBlocks, Parser};
 ///
 /// # Example
 ///
-/// ```ignore
-/// #[cxx_qt::bridge(namespace = "cxx_qt::my_object")]
```

Comment:
oddly Windows gets confused with the namespace

```
  = note: rust_out.rust_out.b333b0d2-cgu.0.rcgu.o : error LNK2019: unresolved external symbol cxx_qt$my_object$cxxbridge1$MyObject$property_changed referenced in function _ZN8rust_out3ffi10MyObjectQt16property_changed17hd775c3ccb607517eE
          rust_out.rust_out.b333b0d2-cgu.0.rcgu.o : error LNK2019: unresolved external symbol cxx_qt$my_object$cxxbridge1$MyObject$rust referenced in function _ZN8rust_out3ffi10MyObjectQt4rust17hfc2e0d9d9b13c063E
          rust_out.rust_out.b333b0d2-cgu.0.rcgu.o : error LNK2019: unresolved external symbol cxx_qt$my_object$cxxbridge1$MyObject$rust_mut referenced in function _ZN8rust_out3ffi10MyObjectQt8rust_mut17h5a53d6213a79e672E
          C:\Users\RUNNER~1\AppData\Local\Temp\rustdoctestvRP7oB\rust_out.exe : fatal error LNK1120: 3 unresolved externals
```

As the other tests don't fail and specifically this one fails

Reply:
Or it might be that the properties internally use `rust`, `rust_mut`, `prop_changed`, but oddly signals do not fail which use `emit_signal` and the `queue` methods which are also `extern "C++"` do not fail :thinking:

Reply:
So having no namespace and no properties has worked, now time to reintroduce one of them to see what happens ...

Reply:
The namespace is fine, it's something odd in the properties, for now lets not put properties in those macro examples that aren't related to properties anyway...